### PR TITLE
zhdocs: translate zh collapsingmergetree.md 

### PR DESCRIPTION
**Pull request by AlexZFX**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

For changelog. Remove if this is non-significant change.

Category :
- Documentation

Short description :
- translate collapsingmergetree.md
- fix zh attention format

### CODE REVIEW

**Comment by blinkov** [comment]
cc @zhang2014 @sundy-li

File: docs/zh/operations/table_engines/collapsingmergetree.md [hunk 1]
```diff
@@ -21,22 +21,22 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 [SETTINGS name=value, ...]
 ```
 
-For a description of request parameters, see [request description](../../query_language/create.md).
```
Comment (zhang2014) [inline docs/zh/operations/table_engines/collapsingmergetree.md]: It is different from https://github.com/yandex/ClickHouse/pull/5168/files#diff-e98acaf76ea56d6ac0c606763b49263cR24. maybe there is no synchronization of en documents before translation?

Comment (AlexZFX) [reply docs/zh/operations/table_engines/collapsingmergetree.md]: I‘m sorry for the negligence, there were five differences but i just found four and synchronized then before. I 'll be more careful next time.
Really thanks for your review.

Comment (AlexZFX) [reply docs/zh/operations/table_engines/collapsingmergetree.md]: It seems that this line does't have any difference。 Or is there anything i haven't noticed?

**Pull request merged**